src/supermarket/app/views/cookbook_versions/index.atom.builder

```diff
@@ -0,0 +1,22 @@
+atom_feed language: 'en-US' do |feed|
+  feed.title 'Cookbook Releases'
+  feed.updated safe_updated_at(@cookbook_versions)
+  feed.link rel_next_prev_link_tags @cookbook_versions
+
+  @cookbook_versions.each do |release|
+    feed.entry release,
+               url: cookbook_version_url(release.cookbook, release.version),
+               published: release.created_at do |entry|
+      entry.title "#{release.name} #{release.version}"
+      entry.content <<~CONTENT, type: 'html'
+        #{release.owner.name}: #{release.name} #{release.version} released by #{release.published_by.name}
```

Comment:
Hey Robb,

I'd change this line to something like: 
`#{release.published_by.name} released version #{release.version} of #{release.name}`

This only shows the publisher of a new cookbook version, removes the author (as this is pretty annoying when you're the only one, who uploads new cookbook versions) and changed the wording a bit.

Reply:
We could make this a little more complicated and show a different message for when the release is the owner versus when the releaser is a collaborator. Like:

### Owner is Releasing

> Sous Chefs: awesome_sauce 1.2.3 released

### Collaborator is Releasing

> Sauce Chefs: awesome_sauce 1.2.3 released by Hot Sauce Fan

What do you think?

Reply:
That sounds like a great idea!
I'd really prefer that. Otherwise no one knows, who released it.